plexapi/base.py

```diff
@@ -489,7 +489,9 @@ class PlexPartialObject(PlexObject):
     }
 
     def __eq__(self, other):
-        return other not in [None, []] and self.key == other.key
+        if isinstance(other, PlexPartialObject):
+            return other not in [None, []] and self.key == other.key
```

Comment:
isn't it superfluous to check `None` and `[]`, as `isinstance(other, PlexPartialObject):` check is already made?

Reply:
Yes, good catch.

Reply:
https://github.com/pkkid/python-plexapi/pull/1230